requirements.txt

```diff
@@ -1,5 +1,5 @@
-dask==2022.9.2
-distributed==2022.9.2
+dask>=2022.10.1
+distributed>=2022.10.1
```

Comment:
Not strictly necessary, but not harmful either, we currently pin `>=2022.9.2` so we rely on the latest Dask/Distributed, only at code freeze we will increase the pin.